check_samples_double_format.sh

```diff
@@ -1,5 +1,6 @@
 #! /bin/bash
```

Comment:
Do you use shfmt and shell check?

https://github.com/mvdan/sh

https://github.com/koalaman/shellcheck

There are GHA for them, they are pretty useful

Reply:
I use shellcheck for work, haven't here